src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/EnvelopeDeletionTest.java

```diff
@@ -103,11 +103,12 @@ private void times(int times, Runnable actionToRun) {
         IntStream.range(0, times).forEach(index -> actionToRun.run());
     }
 
-    private List<ListBlobItem> searchByName(CloudBlobContainer container, String fileName) {
-        Iterable<ListBlobItem> blobs = container.listBlobs(null, true, EnumSet.of(SNAPSHOTS), null, null);
-
-        return stream(blobs.spliterator(), false)
-            .filter(b -> b.getUri().getPath().contains(fileName))
+    private List<BlobItem> searchByName(BlobContainerClient container, String fileName) {
+        ListBlobsOptions listOptions = new ListBlobsOptions();
```

Comment:
`var`? just maybe in general let's go through the pr and add `var`s where makes sense

Reply:
next pr.